docs/source/spawners.md

```diff
@@ -162,7 +162,32 @@ When `Spawner.start` is called, this dictionary is accessible as `self.user_opti
 
 [Spawner]: https://github.com/jupyter/jupyterhub/blob/master/jupyterhub/spawner.py
 
-
 ## Writing a custom spawner
 
 If you are interested in building a custom spawner, you can read [this tutorial](http://jupyterhub-tutorial.readthedocs.io/en/latest/spawners.html).
+
+## Exposing resource limits & guarantees
+
+Some spawners allow setting CPU / Memory limits and / or guarantees on the spawned single-user servers. Such spawners should expose the limits/guarantees to the single-user servers via environment variables.
```

Comment:
Some spawners allow setting CPU limits, memory limits, or guarantees on the single-user notebook servers. Spawners that support any of these settings should expose the limits or guarantees to the single-user notebook servers via environment variables.